packages/desktop-client/src/components/settings/index.js

```diff
@@ -27,10 +28,22 @@ import { Section, AdvancedToggle } from './UI';
 
 function About() {
   const version = useServerVersion();
+  const latestVersion = useLatestVersion();
+  const isOutdated = useIsOutdated();
 
   return (
     <Section title="About" style={{ gap: 5 }}>
-      <Text>Client version: v{window.Actual.ACTUAL_VERSION}</Text>
+      <Text>
+        Client version: v{window.Actual.ACTUAL_VERSION} (
+        {!isOutdated ? (
+          <span>latest</span>
+        ) : (
+          <a href="https://actualbudget.com/release-notes">
+            {'new version available: ' + latestVersion}
+          </a>
+        )}
```

Comment:
Can you swap this conditional so that the condition is `isOutdated`?
```suggestion
        {isOutdated ? (
          <a href="https://actualbudget.github.io/docs/Release-Notes">
            new version available: {latestVersion}
          </a>
        ) : (
          <span>latest</span>
        )}
```